src/components/features/most-popular-section/tabs.tsx

```
@@ -2,6 +2,7 @@ import { Card, CardHeader, CardTitle, CardDescription, CardContent } from "@/com
 import { Tabs, TabsList, TabsTrigger, TabsContent } from "@/components/ui/tabs";
 import ProductCard from "@components/shared/product-card";
```

ESLint

[error] 'ProductCard' is defined but never used.

(@typescript-eslint/no-unused-vars)